### PR TITLE
feat(docs/natives): improve description

### PR DESCRIPTION
**Pull request by PsychoShock**
Still in progress. This PR will have a goal to review a lot of natives missing descriptions.

### CODE REVIEW

**Comment by ahcenezdh** [comment]
A maximum of 8 files can be changed (including adding or deleting files), as suggested by @4mmonium, to make it easier to review

File: SYSTEM/Settimera.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ ns: SYSTEM
 void SETTIMERA(int value);
 ```
 
-
 ## Parameters
-* **value**: 
+* **Time**: In milliseconds.
```
Comment (4mmonium) [inline SYSTEM/Settimera.md:12]: Hey, rename this to `value`, it needs to match the function argument declared on the function declaration.

File: SYSTEM/Settimerb.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ ns: SYSTEM
 void SETTIMERB(int value);
 ```
 
-
 ## Parameters
-* **value**: 
+* **Time**: In milliseconds.
```
Comment (4mmonium) [inline SYSTEM/Settimerb.md:12]: Hey, rename this to `value`, it needs to match the function argument declared on the function declaration.

File: PED/SetPedCombatRange.md [hunk 3]
```diff
@@ -17,5 +17,6 @@ Only the values 0, 1 and 2 occur in the decompiled scripts. Most likely refers d
 
 ## Parameters
 * **ped**: 
-* **p1**: 
+* **combat range**: 0, 1 or 2
```
Comment (4mmonium) [inline PED/SetPedCombatRange.md:20]: Parameter names cannot have spaces, they also need to match the declared parameters on top.

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Hey there, left you some feedback. 
Here's the [contribution guide](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md) for you to use as reference.
Merry Christmas by the way!

File: PED/SetPedCombatRange.md (hunk 3 above)
Comment (Z3rio) [reply PED/SetPedCombatRange.md:20]: yeah, this could probably just be `range` instead.
and an explanation of the variable would probably be beneficial

**Comment by PsychoShock** [comment]
Damn sorry that's a messy PR. I'll make the reste cleaner.

**Comment by PsychoShock** [comment]
This PR should be good to review @4mmonium . Thanks.

File: PED/GivePedHelmet.md [hunk 4]
```diff
@@ -8,19 +8,11 @@ ns: PED
 void GIVE_PED_HELMET(Ped ped, BOOL cannotRemove, int helmetFlag, int textureIndex);
 ```
 
-```
-PoliceMotorcycleHelmet   1024
-RegularMotorcycleHelmet   4096
-FiremanHelmet 16384
-PilotHeadset  32768
-PilotHelmet   65536
```
Comment (4mmonium) [inline PED/GivePedHelmet.md:16]: Thought I would say, the texture helmet indexes are kind of useful to have.

Comment (PsychoShock) [reply PED/GivePedHelmet.md:16]: Should be good.

**Review by 4mmonium — APPROVED** [review]
Looks good to me, merging! Thank you 😊

**Pull request merged**